src/lseq.rs

```diff
@@ -133,71 +141,59 @@ impl<T: Clone, A: Ord + Clone> LSeq<T, A> {
     /// # Panics
     ///
     /// * If the allocation of a new index was not between `ix` and `ix - 1`.
```

Comment:
Does this still panic in such case?

Reply:
It does not! I've gone ahead and updated all the docs